src/validation.js

```diff
@@ -224,21 +235,21 @@ export default class AuroFormValidation {
 
       if (elem.setCustomValidity) {
         elem.errorMessage = elem.setCustomValidity;
-      } else if (elem.tagName.toLowerCase() === 'auro-input') {
+      } else if (elem.tagName.toLowerCase().startsWith('auro-input')) {
         const input = elem.renderRoot.querySelector('input');
 
         if (input.validationMessage.length > 0) {
           elem.errorMessage = input.validationMessage;
         }
-      } else if (this.auroInputElements && this.auroInputElements.length > 0) {
-        const firstInput = this.auroInputElements[0].renderRoot.querySelector('input');
+      } else if (this.inputElements && this.inputElements.length > 0) {
+        const firstInput = this.inputElements[0].renderRoot.querySelector('input');
 
-        if (firstInput.validationMessage.length > 0) {
+        if (firstInput && firstInput.validationMessage.length > 0) {
           elem.errorMessage = firstInput.validationMessage;
-        } else if (this.auroInputElements.length === 2) {
-          const secondInput = this.auroInputElements[1].renderRoot.querySelector('input');
+        } else if (this.inputElements.length === 2) {
+          const secondInput = this.inputElements[1].renderRoot.querySelector('input');
 
-          if (secondInput.validationMessage.length > 0) {
+          if (secondInput && secondInput.validationMessage.length > 0) {
```

Comment:
Logic is incorrect here. You changed this to get the HTML5 input and then look for the HTML5 input inside of the HTML5 input.